src/Statistics/TopScore.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Statistics;
+
+use VSV\GVQ_API\User\ValueObjects\Email;
+
+class TopScore
+{
+    /**
+     * @var Email
+     */
+    private $email;
+    /**
+     * @var int
+     */
+    private $score;
+
+    public function __construct(Email $email, int $score)
+    {
+        $this->email = $email;
+        $this->score = $score;
```

Comment:
Hier is een guard ook mogelijk met een throw van `InvalidArgumentException` als `score` kleiner is dan 0.